### DISCUSSION TITLE
reinstalling server from downloaded image

### DISCUSSION THREAD

**Discussion by dewaste08** [itzg/docker-minecraft-server#2129] [Q&A]
Hi, I had several issues with my NAS so i was forced to reinstall everything, before this I downloaded the image of my minecraft server that was running on it but now I don't know if it's possible to reinstall a server with the exact same world as before, I looked for a long time on the internet and on the github page but I'm unable to find anything, could someone help me please ? That would be much appreciated !

**Comment by itzg** [comment]
I'm not sure what you mean by "downloaded the image". If you had used a host directory mount for `/data`, then simply recreating with the same directory content mounted will get it back to where it was

https://github.com/itzg/docker-minecraft-server#attaching-data-directory-to-host-filesystem

**Reply by dewaste08** [reply]
Thanks for the answer ! i mean i exported the server image via portainer, which downloaded the image in .rar locally on my computer, i then copied the file to a folder on my server

**Reply by itzg** [reply]
Look for the directory that contains the server jar and world directory -- that'll be the one to attach as data volume. There's a chance though that it's export doesn't include volumes, such as `/data`.

**Reply by dewaste08** [reply]
I imported the image on portainer and tried to reinstall the serv with the following tag : `itzg/minecraft-server:latest`, but at the end it was a complete new world created, I think that I lost my previus server.... Im totaly new and discovering everything, it's not a big loss it makes me learn

**Reply by itzg** [reply]
Bummer. For next time definitely make use of host directory attached volume for the container's `/data` volume.
